src/Pux.purs

```diff
@@ -59,11 +59,11 @@ start config = do
       htmlSignal = stateSignal ~> config.view
       mapAffect affect = launchAff $ unsafeCoerceAff do
         ev <- affect
-        later $ case ev of
+        case ev of
```

Comment:
````later```` was replaced with ````delay```` (https://github.com/slamdata/purescript-aff/commit/1ed0d751dd7427074f98d86220566a58572d5aa7)
```delay (Milliseconds 0.0)``` should then be placed above the case